src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -899,41 +899,39 @@ Sentry owner or manager permissions are required to install this integration. Sl
 
 1. In Sentry, navigate to Organization Settings > **Integrations**.
 
-1. At the top of the page, you’ll see the available Global Integrations list, which includes Slack. Click **Install** to integrate Sentry with your Slack workspace(s).
+1. Once on the page, you’ll see the available Global Integrations list, which will include Slack. You can use the search bar to filter for Slack, if it doesn't appear at the top. 
+Click on **Slack** to go to the details page for the integration.
 
-    [{% asset slack-all-integrations-page.png %}]({% asset slack-all-integrations-page.png @path %})
+    [{% asset slack-integration-directory.png %}]({% asset slack-integration-directory.png @path %})
 
 1. Click **Add workspace**.
 
-    [{% asset slack-auth-choose-workspace.png %}]({% asset slack-auth-choose-workspace.png @path %})
+    [{% asset slack-overview-details-page.png %}]({% asset slack-overview-details-page.png @path %})
 
-    You can toggle the Slack workspace you want to connect in the dropdown menu in the upper right corner of the authentication window. To set up multiple workspaces, go through this process for each separate workspace.
 
-1. After you've connected to the correct workspace, click **Continue**.
+1. You can toggle the Slack workspace you want to connect in the dropdown menu in the upper right corner of the authentication window. To set up multiple workspaces, go through this process for each separate workspace. Once you've chosen the correct workspace, click **Allow**.
 
-1. On the next page, you can choose the scope of channels you’d like Sentry to access. You can specify all public channels or choose specific channels. Then click **Install**.
+    [{% asset slack-allow-install.png %}]({% asset slack-allow-install.png @path %})
 
-    [{% asset slack-auth-channel-pref.png %}]({% asset slack-auth-channel-pref.png @path %})
+1. Your integration details page will refresh and show the Slack workspace you just added.
 
-1. Your Organization Integrations page will refresh and show the Slack workspace you just added.
+    [{% asset slack-install-success.png %}]({% asset slack-install-success.png @path %})
 
-    [{% asset slack-add-workspace-success.png %}]({% asset slack-add-workspace-success.png @path %})
-
-    You’ll also see the Sentry app join the Slack channels you specified right away.
-
-    [{% asset slack-sentry-integration-in-channel.png %}]({% asset slack-sentry-integration-in-channel.png @path %})
-
-Now your Slack integration is available to all projects in your Sentry organization. See the next section to configure your notification settings.
+Now your Slack integration is available to all projects in your Sentry organization. In order to get Slack notifications for private channels, please make sure to add the Slack app to the channel. You can do so easily by using `@sentry` to invite the bot to the channel. See the next section to configure your notification settings.
```

Comment:
Suggest: "Your Slack integration is now available to all projects in your Sentry organization. To enable Slack notification for private channels, add the Slack app to the channel. One quick method: use `@sentry`to invite the Sentry bot the Slack channel. 

In the next section, we'll walk you through configuring your notification settings."